From b88bbf020394cd1706ee50cc4563feee7d072ed7 Mon Sep 17 00:00:00 2001
From: Andrew Jones <drjones@redhat.com>
Date: Thu, 18 Sep 2014 23:49:31 +0200
Subject: Revert "rhel: Drop "ivshmem" device"

Message-id: <1411084171-28836-5-git-send-email-drjones@redhat.com>
Patchwork-id: 61306
O-Subject: [PATCH RHEV-7.1 qemu-kvm-rhev 4/4] Revert "rhel: Drop "ivshmem" device"
Bugzilla: 1104063
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Marcel Apfelbaum <marcel.a@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>

This reverts commit 3350f809f674a56c9467343e478b36ea94c4a14a.

Signed-off-by: Andrew Jones <drjones@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>

Conflicts:
	hw/misc/Makefile.objs

diff --git a/hw/misc/Makefile.objs b/hw/misc/Makefile.objs
index 1cc0b76..4aa76ff 100644
--- a/hw/misc/Makefile.objs
+++ b/hw/misc/Makefile.objs
@@ -19,8 +19,7 @@ common-obj-$(CONFIG_PUV3) += puv3_pm.o
 
 common-obj-$(CONFIG_MACIO) += macio/
 
-# Disabled for Red Hat Enterprise Linux:
-# obj-$(CONFIG_IVSHMEM) += ivshmem.o
+obj-$(CONFIG_IVSHMEM) += ivshmem.o
 
 obj-$(CONFIG_REALVIEW) += arm_sysctl.o
 obj-$(CONFIG_NSERIES) += cbus.o
